subliminal/refiners/hash.py

```diff
@@ -10,6 +10,8 @@
     'napiprojekt': hash_napiprojekt,
     'opensubtitles': hash_opensubtitles,
     'opensubtitlesvip': hash_opensubtitles,
+    'opensubtitlescom': hash_opensubtitles,
```

Comment:
are we sure they're using the same hashing algorithm?

Reply:
I think so:
https://opensubtitles.stoplight.io/docs/opensubtitles-api/e3750fd63a100-getting-started#calculating-moviehash-of-video-file